db/re/item_combo_db.txt

```diff
@@ -447,6 +447,7 @@
 5917:1525,{ bonus bLongAtkDef,10; }
 5920:18542,{ bonus bHealPower,2*getequiprefinerycnt(EQI_HEAD_TOP); }
 5967:28321,{ bonus2 bSubSkill,"HT_BLITZBEAT",200; }
+5918:19306,{ bonus bAspd,1; bonus bCritAtkRate,2 + 2 * (readparam(bDex) / 10); bonus2 bSubRace,RC_DemiHuman,2; bonus2 bSubSkill,"NPC_CRITICALSLASH",100; .@luk = readparam(bLuk); if (.@luk > 119) { bonus bAspd,2; bonus bCritAtkRate,4; bonus2 bSubRace,RC_DemiHuman,4; bonus bSplashRange,1; } else if (.@luk > 107) { bonus bAspd,1; bonus bCritAtkRate,2; bonus2 bSubRace,RC_DemiHuman,2; } }
```

Comment:
- Maybe the bonus when the luk is highter than 107 and 119 add up each other ?
- A compact version could be
```
5918:19306,{
.@aspd = 1;
.@crit_dmg = 2 + 2 * (readparam(bDex) / 10);
.@sub_demi = 2;
.@luk = readparam(bLuk);
if (.@luk > 107) { .@aspd += 1; .@crit_dmg += 2; .@sub_demi += 2; } }
if (.@luk > 119) { .@aspd += 2; .@crit_dmg += 4; .@sub_demi += 4; bonus bSplashRange,1; }
bonus bAspd,.@aspd; bonus bCritAtkRate,.@crit_dmg; bonus2 bSubRace,RC_DemiHuman,.@sub_demi; bonus2 bSubSkill,"NPC_CRITICALSLASH",100;
} }
```